client/src/controllers/PreviewController.ts

```diff
@@ -736,10 +739,10 @@ export class PreviewController extends Controller<HTMLElement> {
 
     this.dispatch('loaded', { cancelable: false });
 
-    runContentChecks();
-
-    const onClickSelector = () => this.newTabTarget.click();
-    runAccessibilityChecks(onClickSelector);
+    getAxe().then(() => {
+      runContentChecks();
+      runAccessibilityChecks(() => this.newTabTarget.click());
```

Comment:
They're run in parallel. We need to ensure axe is imported first so we don't import it twice. Not sure how much benefit we get from this – maybe imports are efficient after the first time.